templates/max/README.md

```diff
@@ -0,0 +1,63 @@
+# {{project-name}}
```

Comment:
I generally want to resist having to duplicate information in multiple places. I do like that there is a quick start README here but maybe it should mostly just point to: https://github.com/Cycling74/median/blob/develop/median/README.md#building-externals

and we update the language there to include building from a template?

Reply:
That's would also be a good solution. Maybe you can update the median/README.md file to include the instructions for installing built externals I've added in this README.